internal/operators/ocs/ocs_operator.go

```diff
@@ -98,33 +125,34 @@ func (o *operator) GetMonitoredOperator() *models.MonitoredOperator {
 func (o *operator) GetHostRequirements(_ context.Context, cluster *common.Cluster, host *models.Host) (*models.ClusterHostRequirementsDetails, error) {
 	numOfHosts := len(cluster.Hosts)
 
-	inventory, err := hostutil.UnmarshalInventory(host.Inventory)
-	if err != nil {
-		return nil, err
-	}
-
-	disks := int64(getValidDiskCount(inventory.Disks) - 1)
-
-	if numOfHosts < 3 {
-		return nil, fmt.Errorf("OCS requires a minimum of 3 hosts")
+	var disks int64 = 0
+	if host.Inventory != "" {
```

Comment:
handle the case of empty inventory here too.

Reply:
In compact mode, if the inventory is empty I am returning the requirements with reqDisks as 1 the test case for the same is https://github.com/openshift/assisted-service/blob/c473a5265ab6627d0d6e4b6ce8a725263855f1f4/internal/operators/ocs/ocs_operator_test.go#L111

for minimal/standard mode I am returning the requirements for minimal mode with disks as zero, the test case for this is https://github.com/openshift/assisted-service/blob/c473a5265ab6627d0d6e4b6ce8a725263855f1f4/internal/operators/ocs/ocs_operator_test.go#L187

Reply:
For empty inventory, shouldn't we return 0 for requirements? . @jakub-dzon  @pkliczewski  WDYT?

Reply:
I think that no inventory (missing) is different than value 0.